src/components/ObjectiveRow.vue

```diff
@@ -38,6 +42,10 @@ export default {
       type: Boolean,
       default: false,
     },
+    isLink: {
+      type: Boolean,
+      default: true,
+    },
```

Comment:
Brukes denne noen steder nå? :thinking: Eller er det for fremtidig bruk?

Reply:
Ev. hvis vi skal gjeninnføre (en eller annen) listevisning? Lar den ligge så lenge!